docs/networking.md

```diff
@@ -459,6 +457,14 @@ $ kops create cluster \
 
 You can adjust Cilium agent configuration with most options that are available in [cilium-agent command reference](http://cilium.readthedocs.io/en/stable/cmdref/cilium-agent/).
 
+The following command will create your cluster with desired Cilium configuration
+
+```console
+$ kops update cluster myclustername.mydns.io --yes
+```
+
+##### Enabling logstash
+
 E.g enabling logstash integration would require you to change above block to
```

Comment:
The "above block" reference is now dangling, as a new block got inserted in the middle.

The "E.g" [sic] is also dangling. This reorganization makes the text more confusing.